components/places/src/import/ios/bookmarks.rs

```diff
@@ -228,9 +228,9 @@ fn populate_mirror_tags(db: &crate::PlacesDb) -> Result<()> {
         for item_id in tagged_items {
             log::trace!("tagging {} with {}", item_id, tag);
             db.execute_cached(
-                "INSERT INTO main.moz_bookmarks_synced_tag_relation(itemId, tagId) VALUES(:item_id, :tag_id)",
-                named_params! { ":tag_id": tag_id, ":item_id": item_id },
-            )?;
+                 "INSERT INTO main.moz_bookmarks_synced_tag_relation(itemId, tagId) VALUES(:item_id, :tag_id)",
```

Comment:
(aside: I'm actually quite surprised `cargo fmt` didn't get upset at this specific change, but it seems to be OK with either)